content/en/core/overview/data-flows-for-analytics.md

```diff
@@ -62,6 +62,12 @@ Custom materialized views and functions are added specific to a deployment's nee
 
 Data in the views and functions mentioned in this section is as accurate as the accuracy of the SQL queries. Best practice is to begin the process of defining these objects at design in order to align analytics and dashboards requirements with workflows being deployed.
 
+#### CHT-Sync
+We have built [cht-sync](https://github.com/medic/cht-sync) to be able to sync data from CouchDB to PostgreSQL. It uses logstash and PostgREST to replicate data from CouchDB to PostgreSQL in a more real-time manner. It is a work in progress and is not yet ready for production use. It is currently being used in a few deployments to test its performance and reliability.
```

Comment:
Try to avoid the "we" word as CHT tools are community built and supported. Try something like "cht-sync is able to...".

Also, this is a production ready release so let's drop the beta warning. We _want_ people to use it in prod and will support and fix any bugs they may find. Honestly, I feel more confident about cht-sync than cht-couch2pg at this point!

```suggestion
[cht-sync](https://github.com/medic/cht-sync) is able to sync data from CouchDB to PostgreSQL. It uses logstash and PostgREST to replicate data from CouchDB to PostgreSQL in a real-time manner.
```